app/js/config/environment/dev.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   devHost: 'zhishi-dev.andela.com',
```

Comment:
not really related to your changes, but since auth is being migrated, you don't need this on `zhishi-dev.andela.com`, we needed to set that to make the cookies work initially!